src/NzbDrone.Core/Download/DownloadEventHub.cs

```diff
@@ -40,6 +40,9 @@ public void Handle(DownloadCompletedEvent message)
                 !message.TrackedDownload.DownloadItem.CanBeRemoved ||
                 message.TrackedDownload.DownloadItem.Status == DownloadItemStatus.Downloading)
             {
+                var downloadClient = _downloadClientProvider.Get(message.TrackedDownload.DownloadClient);
+                downloadClient.MarkItemAsImported(message.TrackedDownload.DownloadItem);
```

Comment:
Not sure if this is exactly where this should be. I figured there is not "import marking" if the item is removed from the download client